src/components/SocialMediaLinks/SocialMediaLinks.mdx

```diff
@@ -0,0 +1,77 @@
+import { ArgTypes, Canvas, Description, Meta, Source } from "@storybook/blocks";
+import { GetTableData } from "./SocialMediaLinks";
+import Table from "../Table/Table"
+import Link from "../Link/Link"
+import * as SocialMediaLinksStories from "./SocialMediaLinks.stories";
+
+<Meta of={SocialMediaLinksStories} />
+
+# SocialMediaLinks
+| Component Version | DS Version |
+|-------------------|------------|
+| Added             |  `1.7.2`   |
+| Latest            |  `1.7.2`   |
+
+- [Overview](#overview)
+- [With Controls](#with-controls)
+  - [Available Social Media Types & Their Default Values](#available-social-media-types--their-default-values)
+- [Variations](#variations)
+  - [Layout Variations](#heading-layout)
+  - [Size Variations](#heading-size)
+  - [Color Variations](#heading-color)
+  - [Border Variations](#heading-border)
+  - [Label Variations](#heading-label)
+- [Accessibility](#accessibility)
+- [No linksData Prop (Default)](#no-linksdata-prop-default)
+
+## Overview
+
+<Description of={SocialMediaLinksStories} />
+
+## With Controls
```

Comment:
I did it this way because of a Storybook behavior. In my mind, the default state is simply `<SocialMediaLinks />`, with no passed props. But in order for the controls to display correctly I need to include default values. This results in a somewhat misleading "Show Code" display on the Docs page that includes all the props w/ default values, e.g.:
```
<SocialMediaLInks
    color="textDefault"
    borders="none"
    layout="row"
    showLabels="true"
    size="small"
/>
```
Additionally, it seemed to me, the concept of "With Controls" in this case better matches the more common use case of passing a value to linksData.

Reply:
Ok, understood.  Leave the `With Controls` story as is.  But I would like to highlight the `linksData` info by moving under a separate heading below `Available Social Media Types & Their Default Values`.

Reply:
As you may notice, in the most recent push I moved the info about passing no props into the Variations list. Do you wish me to move this out of that list and move it up as a level 2 subhead, beneath "Available Social Media Types...?"